can-jquery.js

```diff
@@ -11,6 +11,7 @@ var makeArray = require("can-util/js/make-array/make-array");
 var mutate = require("can-util/dom/mutate/mutate");
 var setImmediate = require("can-util/js/set-immediate/set-immediate");
 var canViewModel = require("can-view-model");
+require("./data.js");
```

Comment:
maybe we should not include this right now, and document this module so people can add it.  That way people can experiment with it?  This might be a good way to start.